change/office-ui-fabric-react-2020-09-02-18-45-05-feat-renaming-gridNext.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

This can be `none` since there's no impact on the published package.
```suggestion
  "type": "none",
```